starlette/middleware/wsgi.py

```diff
@@ -43,7 +43,7 @@ def build_environ(scope: Scope, body: bytes) -> dict:
         elif name == "content-type":
             corrected_name = "CONTENT_TYPE"
         else:
-            corrected_name = "HTTP_%s" % name.upper().replace("-", "_")
+            corrected_name = f"HTTP_{name.upper().replace('-', '_')}"
```

Comment:
Let's perhaps pull `.upper().replace('-', '_')` out of the f-string entirely. So we have:

`f"HTTP_{name}".upper().replace("-", "_")`

I think that ends up being cleaner. (I don't really want us to use expressions inside f-strings if we can avoid it.)